src/Confluent.SchemaRegistry/SchemaRegistryConfig.cs

```diff
@@ -80,6 +80,26 @@ public static class PropertyNames
             /// </summary>
             [Obsolete("Subject name strategies should now be configured using the serializer's configuration. In the future, this configuration property will be removed from SchemaRegistryConfig")]
             public const string SchemaRegistryValueSubjectNameStrategy = "schema.registry.value.subject.name.strategy";
+
+            /// <summary>
+            ///     File path to CA certificate(s) for verifying the schema registry's key.
+            /// </summary>
+            public const string SslCaLocation = "schema.registry.ssl.ca.location";
+
+            /// <summary>
+            ///     Ssl keystore location.
+            /// </summary>
+            public const string SslKeystoreLocation = "schema.registry.ssl.keystore.location";
+
+            /// <summary>
+            ///     Ssl keystore password.
+            /// </summary>
+            public const string SslKeystorePassword = "schema.registry.ssl.keystore.password";
+
+            /// <summary>
+            ///     Ssl verify
```

Comment:
This doc string is insufficient, needs to say what it does, and why, and a warning to only disable verification in test/dev environments.